testsuite/utils/src/main/java/org/keycloak/testsuite/KeycloakServer.java

```diff
@@ -400,15 +401,11 @@ public void start() throws Throwable {
 
             di.setDefaultServletConfig(new DefaultServletConfig(true));
 
-            ServletInfo restEasyDispatcher = Servlets.servlet("Keycloak REST Interface", HttpServlet30Dispatcher.class);
-
-            restEasyDispatcher.addInitParam("resteasy.servlet.mapping.prefix", "/");
-            restEasyDispatcher.setAsyncSupported(true);
-
-            di.addServlet(restEasyDispatcher);
+            // Note that the ResteasyServlet is configured via server.undertowDeployment(...);
```

Comment:
@thomasdarimont Are `asyncSupported` and `resteasy.servlet.mapping.prefix` parameter maintained? Perhaps they need to be added similarly as a property of the `deployment`?

Reply:
IMHO yes: https://github.com/resteasy/Resteasy/blob/master/server-adapters/resteasy-undertow/src/main/java/org/jboss/resteasy/plugins/server/undertow/UndertowJaxrsServer.java#L184

Reply:
Perfect, thanks for checking!